Spanner/tests/Unit/ResultTest.php

```diff
@@ -307,7 +307,7 @@ public function testTransaction()
         $fixture = $this->getStreamingDataFixture()['tests'][1];
         $result = $this->getResultClass($fixture['chunks'], 'rw');
 
-        $this->assertNull($result->transaction());
+        $this->assertInstanceOf(Transaction::class, $result->transaction());
```

Comment:
Same as the assertion after `next()` call
```suggestion
```

Reply:
Removed the `next()` and assertion after `next()` as the `transaction`/`snapshot` gets created in the constructor of `Result` itself.